packages/lesmis/src/routes/PageRoutes.js

```diff
@@ -32,14 +30,8 @@ export const PageRoutes = React.memo(() => (
     <Route path="/contact">
       <ContactView />
     </Route>
-    <Route exact path="/:province">
```

Comment:
Now that we have a url design, I am updating the routes to match. URL design was done in [this](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/2274) task